tests/examples/test_examples.py

```diff
@@ -111,6 +156,12 @@ def test_mlflow_run_example(directory, params, tmpdir):
         ("shap", ["python", "regression.py"]),
         ("shap", ["python", "binary_classification.py"]),
         ("shap", ["python", "multiclass_classification.py"]),
+        ("pytorch/MNIST/example1", ["python", "mnist_autolog_example1.py", "--max_epochs", "1"]),
+        ("pytorch/MNIST/example2", ["python", "mnist_autolog_example2.py", "--max_epochs", "1"]),
+        (
+            "pytorch/BertNewsClassification",
+            ["python", "bert_classification.py", "--max_epochs", "1", "--num_samples", "100"],
+        ),
```

Comment:
I think we'd need to remove these if we remove `transformers` and `torchtext` from `extra-ml-requirements.txt`.